### DISCUSSION TITLE
How to visualize a 3D process in IBIP simulation with certain inlet ?

### DISCUSSION THREAD

**Discussion by eromanga233** [PMEAL/porespy#851] [Q&A]
In `ps.visualization.satn_to_movie` method, it seems that porespy does not support 3D pore image. 
It always goes as `TypeError: Invalid shape (64, 64, 64) for image data`, so I think it's the problem of matplotlib.
Since porespy supports 3D image in almost each modules, it's a pity that we can not visualize a 3D process in IBIP simulation with certain inlet.
We wish this requirement to be considered~

**Comment by jgostick** [comment]
If you want to visualize in 3d you need paraview or a commercial package like dragonfly.
